x-pack/plugins/fleet/public/components/platform_selector.tsx

```diff
@@ -50,6 +50,21 @@ export const PlatformSelector: React.FunctionComponent<Props> = ({
     />
   );
 
+  const getCommandByPlatform = (plat: PLATFORM_TYPE) => {
+    switch (plat) {
+      case 'linux':
+        return linuxCommand;
+      case 'mac':
+        return macCommand;
+      case 'windows':
+        return windowsCommand;
+      case 'deb':
+        return linuxDebCommand;
+      case 'rpm':
+        return linuxRpmCommand;
+    }
+  };
+
```

Comment:
Sort of a nit/open question, but what are folks' thoughts on using mapping objects versus `switch` statements, e.g

```ts
const commandsByPlatform: Record<PLATFORM_TYPE, string> = {
  linux: linuxCommand,
  mac: macCommand,
  windows: windowsCommand,
  deb: linuxDebCommand,
  rpm: linuxRpmCommand,
}
```

It's a personal bias, but I always find `switch` statements a little cumbersome. In this case, though, this `switch` is pretty functionally equivalent to doing something with a mapping object as above since we have no `default` case, though, so I don't really know that it's worth changing. Just curious on folks opinions 🙂

Reply:
good point, I like the map better too, as it is shorter, let me update.